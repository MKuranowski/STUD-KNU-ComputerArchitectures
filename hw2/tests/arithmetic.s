addi x20, x0, 819
addi x21, x0, 341
addi x22, x0, -42
addi x23, x0, 45
addi x24, x0, 2
addi x25, x0, 6
add x1, x20, x21
add x1, x1, x22
sub x2, x21, x20
sub x2, x2, x22
sll x3, x23, x25
srl x4, x23, x24
xor x5, x20, x21
and x6, x20, x21
or x7, x20, x21
