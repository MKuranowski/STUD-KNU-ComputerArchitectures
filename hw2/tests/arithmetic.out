Processor's clock cycles: 15
>>>>>>>>[REGISTER DUMP]<<<<<<<
PC: = 60
x00 = 0
x01 = 1118
x02 = -436
x03 = 2880
x04 = 11
x05 = 614
x06 = 273
x07 = 887
x08 = 0
x09 = 0
x10 = 0
x11 = 0
x12 = 0
x13 = 0
x14 = 0
x15 = 0
x16 = 0
x17 = 0
x18 = 0
x19 = 0
x20 = 819
x21 = 341
x22 = -42
x23 = 45
x24 = 2
x25 = 6
x26 = 0
x27 = 0
x28 = 0
x29 = 0
x30 = 0
x31 = 0
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
>>>>>>>>[MEMORY DUMP]<<<<<<<<<
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
